os/drivers/audio/alc5658.c

```diff
@@ -1014,8 +1063,8 @@ static void alc5658_rxtxcallback(FAR struct i2s_dev_s *dev, FAR struct ap_buffer
 
 	alc5658_takesem(&priv->devsem);
 	sq_entry_t *tmp;
-	for (tmp = (sq_entry_t *)sq_peek(&priv->pendq); tmp; tmp = sq_next(tmp)) {
-		if (tmp == (sq_entry_t *)apb) {
+	for (tmp = (sq_entry_t *) sq_peek(&priv->pendq); tmp; tmp = sq_next(tmp)) {
```

Comment:
casting

Reply:
This is modified by our formatter script.